test_vm/vendor/cookbooks/dbfit_test/recipes/diskspace.rb

```diff
@@ -0,0 +1,17 @@
+chef_gem 'di-ruby-lvm'
+
+lvm_physical_volume '/dev/sdb' do
+  action :create
+end
+
+lvm_volume_group 'centos' do
+  action :extend
+  physical_volumes ['/dev/sdb']
+end
+
+lvm_logical_volume 'root' do
+  group 'centos'
+  action :resize
+  size '9G'
+  take_up_free_space true
```

Comment:
I suspect we should have only one of the two size specifications: `size '9G'` and `take_up_free_space true`?